bundles/plugins/org.bonitasoft.studio.expression.editor/src/org/bonitasoft/studio/expression/editor/operation/OperationReturnTypesValidator.java

```diff
@@ -169,6 +173,14 @@ public IStatus validate(final Object value) {
         return ValidationStatus.ok();
     }
 
+    private boolean isInvalidQueryExpression(final Operation operation) {
+        if (!operation.getLeftOperand().getReferencedElements().isEmpty()) {
```

Comment:
operation.getLeftOperand().getReferencedElements() can be extracted
Are we sure that the left Operand is never null?